pegasus/helpers/section_api_helpers.rb

```diff
@@ -185,7 +186,12 @@ def self.valid_courses
            where("hidden = 0").
            select(:id, :name).
            all.
-           map { |c| [c[:id], c[:name]]}
+           map { |c| [c[:id], c[:name]]}.
```

Comment:
and it might be nice to have a comment here since this is a bit of a weird thing to do :)